MonkeyWrench.Web.WebService/WebServices.asmx.cs

```diff
@@ -55,6 +55,21 @@ private void VerifyUserInRole (DB db, WebServiceLogin login, string role, bool @
 			Authentication.VerifyUserInRole (Context, db, login, role, @readonly);
 		}
 
+		private void VerifyUserInRoles(DB db, WebServiceLogin login, string roles, bool @readonly)
+		{
+			string[] r = (roles == null) ? new string[0] : roles.Split(',');
+
+			Authentication.VerifyUserInRoles(Context, db, login, r, @readonly);
```

Comment:
You're not adding `Administrator` to the list here (and you're not calling the overload just below either).